picard/file.py

```diff
@@ -353,16 +354,23 @@ def _move_additional_files(self, old_filename, new_filename):
         patterns = encode_filename(config.setting["move_additional_files_pattern"])
         patterns = filter(bool, [p.strip() for p in patterns.split()])
         for pattern in patterns:
-            # FIXME glob1 is not documented, maybe we need our own implementation?
-            for old_file in glob.glob1(old_path, pattern):
-                new_file = os.path.join(new_path, old_file)
-                old_file = os.path.join(old_path, old_file)
-                # FIXME we shouldn't do this from a thread!
-                if self.tagger.files.get(decode_filename(old_file)):
-                    log.debug("File loaded in the tagger, not moving %r", old_file)
-                    continue
-                log.debug("Moving %r to %r", old_file, new_file)
-                shutil.move(old_file, new_file)
+            pattern_regex = re.compile(fnmatch.translate(pattern), re.IGNORECASE)
+            try:
+                names = os.listdir(old_path)
```

Comment:
`old_path` isn't modified in the loop, please move following lines outside the `for pattern...` loop.
The same goes for `names = filter(...` you may just do it once since `names` doesn't change.
Also in which case does it create an exception ? It shouldn't be hidden imho.